<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EMDtesterClass</class>
 <widget class="QMainWindow" name="EMDtesterClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EMDtester</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>土堆/坑数量：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>土堆各自的量：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>土坑各自的量：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sb_nDim">
    <property name="geometry">
     <rect>
      <x>101</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
   </widget>
   <widget class="QTextEdit" name="t_supplyData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="t_demandData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>401</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_showSdis">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>show supply distribute</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_showDdis">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>140</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>show demand distribute</string>
    </property>
   </widget>
   <widget class="QLabel" name="t_sumS">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>()/()</string>
    </property>
   </widget>
   <widget class="QLabel" name="t_sumD">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>()/()</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_calemd">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cal EMD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_saveTest">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_calGrad">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cal Grad</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>EMD距离：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_emdDis">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>240</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_readSingleData">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>370</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>..</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_readBatchData">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>410</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="t_singleDataPath">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>单个实验数据文件：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>批实验数据文件夹：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="t_batchDataPath">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>410</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_readInSD">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>370</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_readInBD">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>410</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>batch</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>270</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>近似EMD距离：</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_emdDis_ap">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>270</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_calGrad_ap">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cal Grad ap</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_calemd_ap">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cal EMD ap</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_saveTest_ap">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Test ap</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_randomData">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>rand Data</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>421</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="emdtester.qrc"/>
 </resources>
 <connections/>
</ui>
